### DISCUSSION TITLE
cmake not work for me

### DISCUSSION THREAD

**Discussion by calvin2021y** [eclipse-iceoryx/iceoryx2#415] [Q&A]
for linux get error:

```sh
  Compiling iceoryx2 v0.4.1 (/opt/iceoryx2/iceoryx2)
    Finished `release` profile [optimized] target(s) in 13.04s
Built target iceoryx2-build-step
Install the project...
-- Install configuration: "Release"
CMake Error at iceoryx2-ffi/c/cmake_install.cmake:46 (file):
  file INSTALL cannot find
  "/opt/iceoryx2/iceoryx2R/rust/native/release/iceoryx2-ffi-cbindgen/include":
  No such file or directory.
Call Stack (most recent call first):
  cmake_install.cmake:47 (include)

```

try macOS get more error,  last error like "error occurred: Command env -u IPHONEOS_DEPLOYMENT_TARGET".

my  TARGET = Some(x86_64-apple-darwin), MACOSX_DEPLOYMENT_TARGET = Some(10.13).  not sure whare come from the IPHONEOS_DEPLOYMENT_TARGET.

**Comment by elBoberido** [comment]
Can you list all the steps you did in order to get to this error?

**Reply by elBoberido** [reply]
@calvin2021y we would need more info in order to understand your problem
